composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
+      "post-autoload-dump": [
+        "php -r \"symlink('../../../../../../vendor/swagger-api/swagger-ui', './src/Bridge/Symfony/Bundle/Resources/public/swagger-ui');\""
```

Comment:
Yeah, like bower (they still have a `ignore` option in the [`bower.json`](https://github.com/bower/spec/blob/master/json.md#bowerjson-specification))... I think what npm achieves is better in term of deployments but it's another debate. For example, I never received a correct answer on this part of a [recent bash deployment code of mine](http://codereview.stackexchange.com/questions/132844/git-tag-release-build-files-and-reset-to-commit-without):

> is having a tag that refers to an overridden commit a good idea?